roles/atomic_installation_verify/tasks/main.yml

```diff
@@ -0,0 +1,39 @@
+---
```

Comment:
If we are going to use the `rhel7/rsyslog` container, we should probably limit this to just RHEL hosts.

There are some Fedora Dockerfiles that have the `INSTALL` label, so perhaps we should be using those for the Fedora and CentOS cases.  (The CentOS Dockerfiles repo didn't appear to have anything lightweight for this purpose)

https://github.com/fedora-cloud/Fedora-Dockerfiles/search?utf8=%E2%9C%93&q=LABEL+INSTALL